lib/board.rb

```diff
@@ -0,0 +1,67 @@
+require "pry"
+
+class Board
+attr_accessor :cells
+
+  def initialize
+    reset!
+  end
+
+  def reset!
+    @cells = Array.new(9, " ")
+  end
+
+  def display
+    puts " #{@cells[0]} | #{@cells[1]} | #{@cells[2]} "
+    puts "-----------"
+    puts " #{@cells[3]} | #{@cells[4]} | #{@cells[5]} "
+    puts "-----------"
+    puts " #{@cells[6]} | #{@cells[7]} | #{@cells[8]} "
+  end
+
+  def position(input)
+    index = input.to_i-1
+    return @cells[index]
+  end
+
+  def update(input, player)
+    @cells[input.to_i-1] = player.token
+  end
+
+  def valid_move?(input)
```

Comment:
Use 'and' operator (`&&`) to combine multiple logical expressions and utilize their implicit return value.
```
input.to_i.between?(1,9) && !taken(input)
```